File: Programs/Gradebook/Python/Course.py
class Course(object):
    def __init__(self, id, code, name):
        self.id = id
        self.code = code
        self.name = name

    def __repr__(self):
        return repr((self.id, self.code, self.name))

    def getId(self):
        return self.id

    def getCode(self):
        return self.code

    def getName(self):
        return self.name

    def setId(self, id):
        self.id = id

    def setCode(self, code):
        self.code = code

    def setName(self, name):
        self.name = name

File: Programs/Gradebook/Scala/Course.scala
class Course(val newId: Int, val newCode: String, val newName: String){
	var id: Int = newId
	var code: String = newCode
	var name: String = newName

	def getId(): Int = {
		return id
	}

	def getCode(): String = {
		return code
	}

	def getName(): String = {
		return name
	}

	def setId(newId: Int) {
		id = newId
	}

	def setCode(newCode: String) {
		code = newCode
	}

	def setName(newName: String) {
		name = newName
	}
}